app/src/main/res/layout/progress_bar.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="horizontal"
+    android:layout_width="wrap_content"
+    android:layout_height="32dp"
+    android:elevation="1dp"
+    android:background="@drawable/progress_bar_background"
+    android:layout_marginStart="15dp"
```

Comment:
Can this just be layout_margin? You don't need to enumerate each of these.

Reply:
did not know that was a thing lolol i was wondering what it was